projects/microchip/curiosity_pic32mzef/mplab/bootloader/nbproject/configurations.xml

```diff
@@ -190,6 +190,8 @@
       <packs>
         <pack name="PIC32MZ-EF_DFP" vendor="Microchip" version="1.0.20"/>
```

Comment:
can we change all mchp projects to to use DFP version 1.2.51? that's the latest installed by default by mplab.

Reply:
sure!